db/migrate/20250224192149_add_payment_receipt_permission_to_api_keys.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddPaymentReceiptPermissionToApiKeys < ActiveRecord::Migration[7.1]
+  def up
+    ApiKey.update_all(permissions: ApiKey.default_permissions) # rubocop:disable Rails/SkipsModelValidations
+  end
+
+  def down
+    ApiKey.update_all( # rubocop:disable Rails/SkipsModelValidations
+      permissions: ApiKey.default_permissions.without("payment_receipt")
+    )
+  end
```

Comment:
I thought we didn't want to do this because if a keys only have `events:write` for instance, we would modify it and give it all the permissions.